dcm4ceph-core/src/main/java/org/open_ortho/dcm4ceph/util/FileUtils.java

```diff
@@ -169,20 +171,18 @@ public static Properties loadProperties(String conffile) {
     public static Properties loadProperties(File filename) {
         Log.info("Loading Properties file " + filename);
         Properties props = new Properties();
-
-        if (!filename.canRead()) {
-            Log.warn("Cannot read from file " + filename + ".");
-        } else {
-            try {
-                props.load(new FileInputStream(filename));
-            } catch (FileNotFoundException e) {
-                e.printStackTrace();
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
+        try (InputStream is = Files.newInputStream(filename.toPath())) {
+            props.load(is);
+            return props;
+        } catch (IOException e) {
+            e.printStackTrace();
+            Log.warn("Cannot read from file " + filename.toPath() + ".\n" +
+                    "Using ceph_default.properties.\n" +
+                    "Please use 2 files as input: %name%.jpg and %name%.properties \n" +
+                    "The DICOM output file will be created with default properties.\n" +
+                    "You may find example .properties file here: https://github.com/open-ortho/dcm4ceph/blob/master/dcm4ceph-sampledata/B1893F12.properties ");
+            return null;
```

Comment:
Exit after Error message.

Reply:
Done, not explicitly though. Not calling System.exit from utility file, but handling in function caller.